app/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
     implementation "com.github.tiagohm.MarkdownView:library:$rootProject.markDownViewVersion"
     implementation "com.github.mirrajabi:search-dialog:$rootProject.searchDialogVersion"
     implementation "com.github.BeppiMenozzi:Knob:$rootProject.knobLibraryVersion"
+    implementation 'com.github.warkiz.widget:indicatorseekbar:1.2.9'
```

Comment:
Version should be specified at the application gradle file. Check how other libraries have implemented.